### PR TITLE
Added upper classes for excitatory neurons

### LINKED ISSUE #882: [NTR] Near-Projecting glutamatergic neuron
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
Near-Projecting glutamatergic neuron
*in [aligned_aliases file](https://github.com/patrick-lloyd-ray/cell_set_aligned_alias/blob/main/aligned_aliases.csv), it specifcally stats L5/6, but in [Fig 10 of BioRvix paper](https://doi.org/10.1101/2020.10.19.343129), it says only NP 

**Synonyms**
L5/6 NP

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
A near-projecting glutamatergic neuron (located in cortical layer5/6)?. *true regardless with the current data we are using 
PMID:31209381
DOI:10.1101/2020.10.19.343129

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
glutamatergic neuron

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
http://purl.obolibrary.org/obo/UBERON_0000956 - Cerebral Cortex

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**
xref:(ILX:0770161)
Related to #827

### PR DESCRIPTION
**Pull request by shawntanzk**
Added IT-projecting - related to #878 which is the only subclass under this, however it is L2/3-6: possible to just put it as L2/3-6 IT-projecting instead - Need feedback on this @dosumis

Added ET-projecting (related to #879):
Will add specific L5 under this? (note: it is the only one under, so it is possible to put it as L5 ET-projecting) - need feedback on this @dosumis

Added NP (Fixes #882)

Added CT-projecting neurons (Related to #880 with similar issue as above)

Note: currently missing L6b subplate #881 as they arent group by projection in an upper layer - they transcriptomically related to CT but have differential projections to the thalamus or anterior cingulate (https://www.nature.com/articles/s41586-018-0654-5)

### CODE REVIEW

**Comment by dosumis** [comment]
I think useful to keep the general classes.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -2659,6 +2658,12 @@ AnnotationAssertion(rdfs:label rdfs:seeAlso "see also"^^xsd:string)
 #   Object Properties
 ############################
 
+# Object Property: obo:RO_0013001 (has synaptic IO in)
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:2661]: Is 'IO' an acronym? I'm not sure what the requirements are for labels in RO, but typically labels ought not contain acronyms.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -27379,6 +27403,26 @@ AnnotationAssertion(oboInOwl:inSubset obo:CL_4023011 <http://purl.obolibrary.org
 AnnotationAssertion(rdfs:label obo:CL_4023011 "Lamp5 GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4023011 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 <https://identifiers.org/ensembl:ENSMUSG00000027270>)))
 
+# Class: obo:CL_4023012 (near-projecting glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023012 "A glutamatergic neuron located in the cerebral cortex that projects axons locally rather than long distance")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:27408]: Add full-stop. 

Not sure that "long-distance" is the correct contrast with "locally" -- maybe "distantly" works better simply for readability.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:2661]: I've copied it directly from http://purl.obolibrary.org/obo/RO_0013001
have now added synonym to clarify it (similar to above)

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -27379,6 +27404,26 @@ AnnotationAssertion(oboInOwl:inSubset obo:CL_4023011 <http://purl.obolibrary.org
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023012 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023012 "ILX:0770103")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023012 "NP neuron")
+AnnotationAssertion(rdfs:label obo:CL_4023012 "near-projecting glutamatergic cortical neuron")
+SubClassOf(obo:CL_4023012 obo:CL_0000679)
+SubClassOf(obo:CL_4023012 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
+
+# Class: obo:CL_4023013 (corticothalamic-projecting glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023013 "A glutamatergic neuron located in the cerebral cortex that projects to the thalamus")
```
Comment (patrick-lloyd-ray) [inline src/ontology/cl-edit.owl:27419]: Add full-stop.

**Review by patrick-lloyd-ray — COMMENTED** [review]
Small changes.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (patrick-lloyd-ray) [reply src/ontology/cl-edit.owl:2661]: Ah, okay. If it's okay with RO, it's okay with me.

**Comment by shawntanzk** [comment]
@dosumis should we make this mouse specific too? Not sure if it's needed as these aren't based on expression of genes and all

**Review by dosumis — APPROVED** [review]
Species neutral is fine.  They look very general.

**Pull request merged**